pipeline/image.py

```diff
@@ -0,0 +1,56 @@
+from collections import defaultdict
+import os
+import datajoint as dj 
+from skimage import io, util
+
+schema = dj.schema(dj.config['custom']['database.prefix'] + 'image')
+
+# === Useful functions===
+def get_image_paths():
+    paths = dj.config.get('custom', {}).get('image_root_data_dir',None)
+    return paths
```

Comment:
Not a bad approach on creating an image root directory and sourcing it from the config but... we are building a pipeline afterall. Let's go for it! :nerd_face:

The problem with this approach is that our pipeline is relying on a step that is taken place outside of the pipeline when it doesn't necessary have to; here that is you manually downloading the images to a local directory. Really, we can have our `Imported` table directly source from the web and load our images into the pipeline through some web scraping. I'd recommend thinking first on what your upstream tables could be for this and then having a look at [beautifulsoup4](https://pypi.org/project/beautifulsoup4/) (great python-based webscraping utility).